src/components/search/History/SearchHistory.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Icon as UIIcon, List as UIList } from '@faststore/ui'
+import Button from 'src/components/ui/Button'
+import './SearchHistory.scss'
+import Link from 'src/components/ui/Link'
+import Icon from 'src/components/ui/Icon'
+import useSearchHistory from 'src/sdk/search/useSeachHistory'
+import { formatSearchState, initSearchState } from '@faststore/sdk'
+
+interface SearchHistoryProps {
+  onClear: () => void
+}
+
+const doSearch = (term: string) => {
+  const { pathname, search } = formatSearchState(
+    initSearchState({
+      term,
+      base: '/s',
+    })
+  )
+
+  return `${pathname}${search}`
+}
+
+const SearchHistory = ({ onClear }: SearchHistoryProps) => {
+  const { searchHistory } = useSearchHistory()
+
+  return (
+    <section data-store-search-history>
+      <div data-store-search-history-header>
+        <h4 data-store-search-history-title>History</h4>
+        <Button variant="tertiary" onClick={onClear}>
+          Clear
+        </Button>
+      </div>
+      <UIList variant="ordered">
+        {searchHistory.map((item, index) => (
+          <li data-store-search-history-item key={index}>
+            <Link variant="display" to={doSearch(item)}>
```

Comment:
I forgot what we concluded on this, but I remember mentioning that the arrow-up-right icon is usually an indicator of the page opening in a new tab. If we're going this way, does it make sense to add the following?

```suggestion
            <Link variant="display" to={doSearch(item)} target="_blank">
```

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target

Reply:
Sure!